milestones/ovirt-4.5.4.conf

```diff
@@ -16,11 +16,13 @@ name = oVirt Ansible Collection
 previous = 2.4.0-1
 current = 3.0.0-1
 
-[ovirt-imageio]
-baseurl = https://github.com/oVirt/
-name = ovirt-imageio
-previous = v2.4.6
-current = v2.4.7
+# [ovirt-imageio]
+# baseurl = https://github.com/oVirt/
+# name = ovirt-imageio
+# previous = v2.4.6
+# current = v2.4.7
+# v2.4.7 tag is broken
```

Comment:
Tag is now created properly: https://github.com/oVirt/ovirt-imageio/releases/tag/v2.4.7

Reply:
Will push a separate patch to bump the ovirt-imageio again to v2.4.7